### ISSUE TITLE
[3.5] vh units don’t work as expected in preview

### ISSUE THREAD

**Issue by carlcs** [craftcms/cms#6388]
### Description

Changes made to preview in https://github.com/craftcms/cms/commit/e52e8bd846f57aefbd83757aba755b129aab8052 introduced issues when vh units are used on the page.

Using the following HTML you’d expect the pink section’s height to be 50% of the height of the viewport, but it’s always ~1000px tall.

```twig
<html lang="en">
<body>
    <section style="min-height: 50vh; background: pink"></section>
    <section style="min-height: 1000px; background: lavender"></section>
</body>
</html>
```

### Additional info

- Craft version: 3.5.0-beta.3

**Comment by brandonkelly** [comment]
That sortof makes sense, since `vh` is going to be based on the size of the _viewport_, not the window. As of that commit, the preview iframe is sized based on the height of its content (using [iFrame Resizer](http://davidjbradshaw.github.io/iframe-resizer/)), and the iframe’s parent div is actually the thing that has `overflow: auto` now. So the iframe viewport is potentially much taller than the window, hence `50vh` resolving to be taller than 50% of the window height.

The change enabled us to improve support for maintaining the scroll position between refreshes, as well as fix #5547, so not interested in reverting it, but maybe it’s worth at least making it optional.

**Comment by brandonkelly** [comment]
I’ve just added a new `previewIframeResizerOptions` config setting for the next release, which lets you override the [iFrame Resizer options](http://davidjbradshaw.github.io/iframe-resizer/#options) that are used, or you can set it to `false` to disable iFrame Resizer comletely.

```php
'previewIframeResizerOptions' => false,
```

**Comment by brandonkelly** [comment]
We released Craft 3.5.5 today, which reverts the default Live Preview behavior back to how it was before Craft 3.5, and adds a new [useIframeResizer](https://craftcms.com/docs/3.x/config/config-settings.html#useiframeresizer) config setting that you can set to `true` if you prefer the new behavior (#6645).
